spec/features/preferences/preference_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# # frozen_string_literal: true
+
+# require 'rails_helper'
+
+# describe 'Preferences' do
+#   describe 'when logged in' do
+#     let(:user) { create(:user) }
+#     let!(:preference) { create(:preference, user_id: user.id) }
```

Comment:
Be careful with this extra preference you are creating here, maybe remove this one and adjust expectation below for second page.

Reply:
That sounds perfect, I'll definitely do it!